sdks/python/apache_beam/testing/test_utils.py

```diff
@@ -23,6 +23,13 @@
 
 from apache_beam.utils import retry
 
+
+__all__ = [
+    'compute_hash',
```

Comment:
What is the bar we are using for "public" and inclusion in __all__?  This is public to this module in the sense that tests can use this, but likely doesn't need to be public to users.